src/main/python/verification/controllers.py

```diff
@@ -94,7 +94,22 @@ def verify_id():
     logger.info('Distance: ' + str(distance))
 
     # Extract text
+    # Grab additional parameters specifying techniques
     preferences = {}
+
+    if 'blur_technique' in request.form:
```

Comment:
Make a note, I would like to redo these if-structures in order to reduce the amount of lines to 3 (or even 1) line(s).